components/automate-ui/src/app/pages/+compliance/+profile/+profile-details/profile-details.component.scss

```diff
@@ -6,6 +6,20 @@ chef-loading-spinner {
   width: 100px;
 }
 
+chef-page-header {
+  chef-subheading {
+    max-width: 65%;
+  }
+  
+  .header-buttons-container {
```

Comment:
These are essentially the same styling as the header-buttons slot element, with a higher top pixel, so that it matches what is generally used across the site.